src/api/app/assets/stylesheets/webui/application/dialog.scss

```diff
@@ -40,6 +40,17 @@
       font-weight: bold;
       font-size: 1.2em;
     }
+    .close_dialog {
```

Comment:
`#remove_dialog` cannot be used twice, since it is an ID and we already have a dialog on this page. Even if I use it, it wouldn't help, since all dialogs that have the ID `remove_dialog`, will get removed from the DOM when the `Close` button in a dialog is clicked.

If it gets deleted from the DOM, we cannot open it again. And since you asked me to get rid of the Ajax request, there is no way to do it differently, besides:
* Adjusting the JS, so that clicking the `Close` link in a dialog doesn't remove the dialog from the DOM (this is not really good since in all other cases we have on different pages, this is required)
* Add an AJAX request and loading the dialog into the DOM again and again... (You didn't want this)
* Or as I implemented it: Add a functionality for just showing/hiding dialogs without removing it from the DOM